pkg/pillar/build-kubevirt-dev.yml

```diff
@@ -0,0 +1,33 @@
+# linuxkit build template
+#
+# Copyright (c) 2024 Zededa, Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+---
+org: lfedge
+image: eve-pillar-dev
+config:
+  binds:
+    - /lib/modules:/lib/modules
+    - /dev:/dev
+    - /etc/resolv.conf:/etc/resolv.conf
+    - /run:/run
+    - /config:/config
+    - /:/hostfs
+    - /persist:/persist:rshared,rbind
+    - /usr/bin/containerd:/usr/bin/containerd
+  net: host
+  capabilities:
+    - all
+  pid: host
+  rootfsPropagation: shared
+  devices:
+    - path: all
+      type: a
+  security_opt:
+    - seccomp:unconfined
+  ports:
+    - 2345:2345
```

Comment:
I found out that this part of the config does nothing. Correct me if I'm wrong (@deitch)

We have the same [here](https://github.com/lf-edge/eve/blob/434b0e5f984e8ace9e4f9ffc59c27bc842a6cea7/pkg/pillar/build-dev.yml#L29-L30) and the port mapping is not part of the final container spec, when the container is running.

However this doesn't matter since we only have one network namespace on EVE, so ports are shared between the host and containers.

Reply:
surprisingly, no. but maybe it's because the network namespace is share due to `net: host` option set above

Reply:
Oh, that is a very good point!

Reply:
@europaul thanks for the review, I removed the port definition.